include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -1033,7 +1033,7 @@ struct __parallel_or_tag
 {
     using _AtomicType = int32_t;
 
-    using FoundLocalReduceOp = sycl::bit_or<_AtomicType>;
+    using FoundLocalReduceOp = __dpl_sycl::__plus<_AtomicType>;
```

Comment:
Did you verify the performance of the addition versus bitwise or? The ladder might be faster.

Reply:
I checking perf now...

Reply:
..sent results to you offline.